app/src/main/java/grinnell/appdev/edu/lyles/fragments/ScheduleFragment.java

```diff
@@ -1,22 +1,82 @@
 package grinnell.appdev.edu.lyles.fragments;
 
+import android.app.ListFragment;
+import android.database.DataSetObserver;
 import android.os.Bundle;
 import android.support.annotation.Nullable;
 import android.support.v4.app.Fragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import java.util.ArrayList;
 
 import grinnell.appdev.edu.lyles.R;
+import grinnell.appdev.edu.lyles.ScheduleItem;
+import grinnell.appdev.edu.lyles.ScheduleTabAdapter;
 
 /**
  * Created by Mattori on 5/9/16.
```

Comment:
Update this documentation to reflect what the class is supposed to be doing. See this for decent documentation examples: https://github.com/GrinnellAppDev/DB-Android/blob/develop/app/src/main/java/edu/grinnell/appdev/grinnelldirectory/models/User.java